### PR TITLE
Added new pools on ETH,BSC,Polygon

### PR DESCRIPTION
**Pull request by Sagar133**

### CODE REVIEW

File: src/static/js/arbitrum_helpers.js [hunk 1]
```diff
@@ -35,7 +35,7 @@ const ArbitrumTokens = [
 const uniSqrtPrice = (tokenDecimals, sqrtRatioX96) => {
   const [token0Decimals, token1Decimals] = tokenDecimals;
   const scalarNumerator = ethers.BigNumber.from(10).pow(token0Decimals);
-  const scalarDenominator = ethers.BigNumber.from(10).pow(token1Decimals);
+  const scalarDenominator = ethers.er.from(10).pow(token1Decimals);
```
Comment (Dimosgr) [inline src/static/js/arbitrum_helpers.js:38]: Hi, you have a typo here

Comment (senamakel) [reply src/static/js/arbitrum_helpers.js]: This has been resolved in the recent commits; do check?

**Comment by senamakel** [comment]
We've fixed the typos and updated it with the latest branch from master

File: src/static/js/arbitrum_helpers.js (hunk 1 above)
Comment (Dimosgr) [reply src/static/js/arbitrum_helpers.js]: ok now its fine thanks

**Pull request merged**